biz.aQute.bnd.runtime/bnd.bnd

```diff
@@ -0,0 +1,29 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
+
+-buildpath: \
+    osgi.annotation;version=latest, \
+    aQute.libg;version=latest;maven-scope=provided;packages=*,\
+    osgi.core;version=@6;maven-scope=provided,\
+    osgi.cmpn;version=@6;maven-scope=provided,\
```

Comment:
Please DO NOT USE osgi.cmpn. I put some effort into ridding the build of osgi.cmpn. Use the individual jars so we have proper control over the versions of individual components used.

Thanks.